### DISCUSSION TITLE
【求助】

### DISCUSSION THREAD

**Discussion by Hejin1212** [share121/inter-knot#2153] [其他]
为什么我看有些帖子的图片是糊的

**Comment by nyakoishi** [comment]
只是因为图片的像素太高需要时间加载吧喵，楼主要不要升级一下网速

**Comment by Dave-12138** [comment]
~因为我把你眼镜摘了~<img src="https://bbs-static.miyoushe.com/static/2024/05/10/0d03e0238a49d7b1e99014ea76cf45ce_6512157066528462204.png" alt="知更鸟 暗中观察" width="75" >
因为加了滤镜，加载中的图片、加载完被判定为nsfw的图片会模糊
